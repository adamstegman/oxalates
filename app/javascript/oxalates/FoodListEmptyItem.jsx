import React from 'react';
import PropTypes from 'prop-types';

import './FoodListEmptyItem.scss';

export class FoodListEmptyItem extends React.Component {
  render() {
    return (
      <li className="food-list-item">
        No foods found
      </li>
    );
  }
};

FoodListEmptyItem.propTypes = {
};
