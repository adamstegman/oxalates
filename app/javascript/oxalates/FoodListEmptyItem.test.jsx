import React from 'react';
import renderer from 'react-test-renderer';

import { FoodListEmptyItem } from './FoodListEmptyItem';

test('FoodListEmptyItem renders an empty list item', () => {
  const component = renderer.create(
    <FoodListEmptyItem />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
