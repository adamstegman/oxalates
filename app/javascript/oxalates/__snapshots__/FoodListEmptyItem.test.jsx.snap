// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FoodListEmptyItem renders an empty list item 1`] = `
<li
  className="food-list-item"
>
  No foods found
</li>
`;
